app/src/main/java/fr/free/nrw/commons/category/CategoriesModel.kt

```diff
@@ -1,11 +1,11 @@
 package fr.free.nrw.commons.category
 
 import android.text.TextUtils
+import fr.free.nrw.commons.category.models.Category
```

Comment:
Well, this is also a model class. Shouldn't we also put this into the model package too? @ben-mathu